src/cares_wrap.cc

```diff
@@ -328,10 +386,18 @@ class QueryWrap : public AsyncWrap {
     return static_cast<void*>(this);
   }
 
+  static void AresQuery(Environment* env, const char* name,
+      int dnsclass, int type, ares_callback callback,
```

Comment:
I realize the code around this doesn’t do it either, but once more, would you mind aligning the arguments vertically? :smile: